app/src/main/java/com/iic/shopingo/ui/trip_flow/fragments/ShoppingListFragment.java

```diff
@@ -0,0 +1,113 @@
+package com.iic.shopingo.ui.trip_flow.fragments;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import com.iic.shopingo.ui.trip_flow.data.ShoppingList;
+import com.iic.shopingo.ui.trip_flow.views.ShoppingListItem;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by asafg on 05/03/15.
+ */
+public class ShoppingListFragment extends Fragment implements ShoppingListItem.OnCallListener {
+
+  private ListView listView;
+  private ShoppingListAdapter adapter;
+  private List<ShoppingList> shoppingLists;
+
+  public ShoppingListFragment() {
+    shoppingLists = new ArrayList<>();
+  }
+
+  @Override
+  public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+    listView = new ListView(getActivity());
+    adapter = new ShoppingListAdapter(getActivity());
+    adapter.setOnCallListener(this);
+    for (ShoppingList sl : shoppingLists) {
+      adapter.addShoppingList(sl);
+    }
```

Comment:
why do you need this code here, you don't have any shopping lists when the view is created and I don't see you retrieving them from the savedInstanceState or some other place.